src/lib/error/ErrorReport.ts

```diff
@@ -140,7 +144,29 @@ export class ErrorReport extends Error {
       stack: this.stack,
       env: this.env,
       ...(this.originalError.code ? { code: this.originalError.code } : null),
+    })
+  }
+
+  private truncateStringsFromObject(element: any, maxStrSize: number = ErrorReport.MAX_ERROR_STRING_LENGTH) {
+    if(element === null || element === undefined) {
```

Comment:
This can be `element == null` - it checks both when `==` is used instead of `===`